python/simple_utils.py

```
@@ -0,0 +1,24 @@
+
+def count_words(sentence): return len(sentence.split())
+
+def celsius_to_fahrenheit(celsius): return (celsius * 9/5) + 32
+
+def calculate_average(numbers):
+return sum(numbers) / len(numbers)
+
+def parse_user_input(data): parts = data.split(",")
+return {"name": parts[0], "age": int(parts[1]), "email": parts[2]}
+
+class DataProcessor: def __init__(self):
```

Ruff (0.15.7)

[error] Multiple statements on one line (colon)

(E701)